recipes/libgcc/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: libgcc
+  version: 5.2.0
```

Comment:
Also, don't we need to be adding gcc as a build requirement or am I missing something?